core/src/main/scala/sttp/tapir/server/package.scala

```diff
@@ -7,5 +7,5 @@ package object server {
     *
     * See also [[DecodeFailureHandling]] and [[DefaultDecodeFailureHandler]].
     */
-  type DecodeFailureHandler = DecodeFailureContext => DecodeFailureHandling
+  type DecodeFailureHandler = (DecodeFailureContext, Endpoint[_, _, _, _]) => DecodeFailureHandling
```

Comment:
can the endpoint be part of `DecodeFailureContext`?

Reply:
sure, done.